index.html

```diff
@@ -12,7 +12,133 @@
+        <li><a href=""><img src="./assets/twitter.svg" alt=""></a></li>
+        <li><a href="">English</a></li>
+      </ul>
+    </div>
+    <nav class="container-nav d-flex flex-direction-row px-2 py-2 ">
+      <div class="nav-log pt-2">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <a class="d-flex" href="index.html"><img class="logoHeader" src="./assets/logo.svg" alt="logo"></a>
+      <ul class="nav-menu ">
+        <li class="page-parts about-link"><a class="nav-link" href="./aboutme.html">About</a></li>
+        <li class="page-parts"><a class="nav-link" href="#program">Program</a></li>
+        <li class="page-parts"><a class="nav-link" href="#blended-ways">Blended Ways</a></li>
+        <li class="page-parts"><a class="nav-link" href="#speakersId">Speakers</a></li>
+        <li class="page-parts"><a class="nav-link" href="#sponsonrs">Partners</a></li>
+        <li class="page-parts campain-link"><a class="nav-link" href="https://thesocialsciences.com/">Campain</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <!-- Main Section -->
+  <main class="page-development d-flex flex-column">
+    <div class="main-titles d-flex flex-column-reverse px-1">
+      <h1 class="main-title">Agency in an Era of Displacement and Social Change</h1>
+      <h2 class="main-subtitle">18th International Conference on Interdisciplinary Social Sciences</h2>
+    </div>
+    <div class="main-description px-2">
+      <p id="program">The conference program groups together presentations along similar themes to facilitate knowledge sharing and community building. The first schedule of sessions will be announced two months prior to the conference start date. In our blended conference model, we will offer both place-based and online options for viewing and engaging</p>
+    </div>
+    <div class="main-location">
+      <h3>2023.7.19(WED)- 21(FRI)</h3>
+      <h6>@ Oxford Brookes University, Oxford, UK + Online</h6>
+    </div>
+  </main>
+
+  <!--Development section of the page-->
+
+  <section class="cards-deployment d-flex flex-column">
+    <div class="blended-title d-flex flex-column align-items-center">
+      <h2 id="blended-ways">Blended Ways of Speaking</h2>
+      <img src="./assets/line.svg" alt="horizontal desing line">
+    </div>
+    <div class="d-flex cards-blended align-items-center">
+      <div class="blended-ways">
+        <img src="./assets/lecture.svg" alt="">
+        <h4>Plenary Sessions</h4>
+        <p>Plenary speakers, chosen from among the world’s leading thinkers, offer formal presentations on topics linked to annual themes.</p>  
+      </div>
+      <div class="blended-ways">
+        <img src="./assets/CCexhibition.svg" alt="">
+          <h4>Garden Conversation</h4>
+          <p>Garden Conversations are open, informal sessions that allow Research Network Members a chance to meet Plenary Speakers.</p>
```

Comment:
- [ ] **[OPTIONAL]** Great job implementing the changes required! 🤓 It would be great if you can set the text-align to center, so it matches more the Behance design, I'm sorry that the previous reviewer didn't point this out! 🤓 🔧